jupyterlab/handlers/announcements.py

```diff
@@ -226,32 +230,54 @@ async def get(self):
                 tree = ET.fromstring(response.body)  # noqa S314
 
                 def build_entry(node):
-                    return Notification(
-                        message=node.find("atom:title", xml_namespaces).text
-                        + "\n"
-                        + node.find("atom:summary", xml_namespaces).text,
-                        createdAt=datetime.fromisoformat(
-                            node.find("atom:published", xml_namespaces).text
-                        ).timestamp()
-                        * 1000,
-                        modifiedAt=datetime.fromisoformat(
-                            node.find("atom:updated", xml_namespaces).text
-                        ).timestamp()
-                        * 1000,
+                    def get_xml_text(attr: str, required: bool = False) -> str:
```

Comment:
Could we do a slight change to mimic dictionary `get` method: `get(key: str, default: Optional[str] = None) -> str` if default is `None`, raise otherwise return default.